### PR TITLE
Comments for All

### PR DESCRIPTION
**Pull request by bugeybot**
This commit mainly serves as a mass cleanup for the code including comments, removing issues, and reformatting for consistency. There is also a small addition of a new folder labeled "Vision" which includes some programs dedicated to handling and storing lines found by cameras. This is untested and even unfinished in its current state, but the code is unused and does build which means it will not and could not affect the robot.

### CODE REVIEW

**Review by BenWertz — APPROVED** [review]
Sort of unrelated, but in the future maybe the `@author` tag should be put there by the person who writes it, instead of whoever goes through and adds comments accrediting themselves. Maybe something to add to CONTRIBUTING.md, saying every new program needs a block comment at the top with at the very least a description and the author's name?

File: src/main/java/frc/robot/Main.java [hunk 1]
```diff
@@ -1,10 +1,3 @@
-/*----------------------------------------------------------------------------*/
```
Comment (JoshRobey) [inline src/main/java/frc/robot/Main.java:1]: I think you need to leave the FIRST copyright

Comment (JoshRobey) [reply src/main/java/frc/robot/Main.java:1]: Just realized that this was in a lot of files.  How about leaving the FIRST copyright and adding a 1073 copyright header?  Maybe something like:
All changes from FIRST/WPLIB original code is copyright FRC Team 1073 and is covered by the same licensing as the FIRST BSD license

File: src/main/java/frc/robot/commands/CargoControls.java [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
 import edu.wpi.first.wpilibj.command.Command;
 import frc.robot.OperatorMode;
 import frc.robot.Robot;
 
+/**
+ * This is the cargo manipulator controls during tele-op.
+ * It is the default command of the cargo subsystem.
+ * 
+ * This command requires the cargo subsystem so that it
+ * take priority over other commands using the subsystem.
```
Comment (JoshRobey) [inline src/main/java/frc/robot/commands/CargoControls.java:12]: take**s** priority

**Comment by JoshRobey** [comment]
First, let me reiterate how important commenting is.  When new students look at this code next year, or even the same students who wrote just a year later, the comments will really help them to understand how the code works.  That should let us build up abilities year after year rather than starting fresh each year.
I also agree with Ben on attribution.  It should include everyone who contributed.

**Comment by JoshRobey** [comment]
And I still can't find the button to approve a review :-(

**Review by 122004 — APPROVED** [review]
Hooray for comments! I agree with both Mr. Robey and Ben, we should add in the CONTRIBUTING.md that the author should always announce themselves unless given discreet permission, and that some sort of license needs to apply to our code. Although the FIRST Copyright may (or may not) be the way to go, we should still consider having something like that. Other than that, looks good :+1:

**Pull request merged**